README.md

```diff
@@ -211,6 +211,7 @@ OPTIONS:
     -p, --package <PKGID>         Package id of the crate to change the version of
     -V, --version                 Print version information
         --workspace               Modify all packages in the workspace
+        --workspace-version       Modify the version set in [workspace.package] (Cargo 1.64.0+)
```

Comment:
I would rather we not do this.  If the user specifies a package that is in the workspace, we then update this field.

In the future, let's discuss design things like this in the issue before moving too far on the PR

Reply:
I did it like that to allow the conflict with `exclude` (since version inheritance can have consequences on even excluded crate), but I can merge with `--workspace` or `--package` where the package inherits the workspace version, see this https://github.com/killercup/cargo-edit/pull/814#issuecomment-1262187887